### PR TITLE
3.0.x new funx

### PR DESCRIPTION
**Pull request by smutlord**
Thank you so much for contributing code to Lando! If this is your **first time** contributing to Lando we recommend you first check our [Getting Involved](https://docs.lando.dev/contrib/contributing.html) docs.

It may also be useful to check the below documentation based on what you are trying to do:

* [Adding code to Lando](https://docs.lando.dev/contrib/contrib-intro.html)
* [Updating documentation](https://docs.lando.dev/contrib/contrib-intro.html)
* [Working on DevOps](https://docs.lando.dev/contrib/contrib-intro.html)
* [Working on Lando's auxiliary services eg websites, apis, etc](https://docs.lando.dev/contrib/contrib-intro.html)
* [Writing Lando Guides](https://docs.lando.dev/contrib/guides-intro.html)
* [Writing Lando blog posts](https://docs.lando.dev/contrib/blogging-intro.html)

We will get to your PR as soon as we can but in the meantime you might want to check to make sure you have done the following. **The more boxes you can check the easier it will be for us to merge in your changes with confidence**

- [ ] My code includes the latest code from the `master` branch
- [ ] My code includes an update to the [CHANGELOG](https://github.com/lando/lando/tree/master/docs/help)
- [ ] My code includes a [functional test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [ ] My code includes a [unit test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [ ] My code includes documentation updates if applicable.
- [ ] My code passes relevant CI status checks
- [ ] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

### CODE REVIEW

File: docs/help/2020-changelog.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Otherwise, scroll down and use the first release that has the green `Latest Rele
 * Added `none` as a Release Channel option [#1894](https://github.com/lando/lando/pull/1894) [#1906](https://github.com/lando/lando/pull/1906) [#2201](https://github.com/lando/lando/pull/2201) [#2240](https://github.com/lando/lando/pull/2240)
 * Added a [guide](https://docs.lando.dev/guides/installing-php-extensions-on-lando.html) on how to install `php` extensions [#2218](https://github.com/lando/lando/pull/2218)
 * Added a [guide](https://docs.lando.dev/guides/guides/installing-node-in-your-lando-php-service.html) on how to install `node` directly in a PHP service for things like PatternLab or Emulsify [#2196](https://github.com/lando/lando/pull/2196)
+* Added support for `mongodb` `4.2` [#2246](https://github.com/lando/lando/pull/2247)
 * Fixed bug where `cache: none` in the Laravel recipe config caused a start error [#2236](https://github.com/lando/lando/pull/2236)
```
Comment (rikvanderkemp) [inline docs/help/2020-changelog.md:15]: Does the changelog need to mention support for 4.0 gets dropped?

Comment (smutlord) [reply docs/help/2020-changelog.md:15]: @rikvanderkemp good call!

**Comment by smutlord** [comment]
@labboy0276 @serundeputy @AaronFeledy the Drupal 9 stuff is in here and the tests are passing but i'd love to get your feedback/review on the docs and other relevant things before i merge.

Note that there is also some mongodb related stuff in here that you can ignore.

File: docs/config/drupal8.md [hunk 2]
```diff
@@ -354,8 +354,8 @@ lando php               Runs php commands
 **Usage examples**
 
 ```bash
-# Download a dependency with drush
-lando drush dl views
+# Doing a drush site install
+lando drush si --db-url=mysql://drupal8:drupal8@database/drupal8 -y
```
Comment (AaronFeledy) [inline docs/config/drupal8.md:358]: `lando drush si --db-url=mysql://drupal9:drupal9@database/drupal9 -y`

**Comment by serundeputy** [comment]
Just read the docs; looks good to me.

**Pull request merged**